index.js

```diff
@@ -23,8 +21,10 @@ if (context.eventName === 'pull_request') {
     return;
 }
 
-try {
-    const changes = JSON.parse(fs.readFileSync(tfplanJsonFile)).resource_changes;
+const inputFilenames = core.getMultilineInput('json-file');
```

Comment:
I guess this answers my previous question - the multiline string approach is probably fine if this helper exists.